### PR TITLE
More faster for cones

### PR DESCRIPTION
**Pull request by admlvntv**

### CODE REVIEW

File: src/main/java/frc/robot/commands/intakerelease/ReleaseCMD.java [hunk 1]
```diff
@@ -3,13 +3,24 @@
...
+import frc.robot.commands.arm.ArmHomeCMD;
 import frc.robot.subsystems.arm.Arm;
 import frc.robot.subsystems.intakerelease.IntakeRelease;
 
 public class ReleaseCMD extends SequentialCommandGroup {
   public ReleaseCMD(IntakeRelease intakerelease, Arm arm) {
-    addCommands(
-        new ArmDropCMD(intakerelease::haveCone, intakerelease::wantsCone, arm),
-        Commands.run(intakerelease::release, intakerelease).withTimeout(0.5));
+    if (intakerelease.wantsCone()) {
```
Comment (elleryjh) [inline src/main/java/frc/robot/commands/intakerelease/ReleaseCMD.java:12]: This wantsCone() check will run at construction of ReleaseCMD() which happens in RobotContainer's constructor.

I think you can do this with 
```
addCommands(
   new ConditionalCommand(coneCommand, cubeCommand, intakerelease::wantsCone));
```

**Comment by admlvntv** [comment]
TODO format (@elleryjh if you could just run spotlessApply and commit it as “format” that would be great)

File: src/main/java/frc/robot/commands/intakerelease/ReleaseCMD.java (hunk 1 above)
Comment (elleryjh) [reply src/main/java/frc/robot/commands/intakerelease/ReleaseCMD.java]: Code looks good now, but I think we should test it out before merging.

**Comment by elleryjh** [comment]
LGTM

**Review by elleryjh — APPROVED** [review]

**Pull request merged**